2015-12-08 10:19:38,131 - streamparse.storm.component - ERROR - Exception in WordCounter.run()
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/streamparse/storm/component.py", line 489, in run
    self._run()
  File "/usr/lib/python2.7/site-packages/streamparse/storm/bolt.py", line 219, in _run
    self.process(tup)
  File "bolts/wordcount.py", line 45, in process
    conn.commit()
NameError: global name 'conn' is not defined
